list.js

```diff
@@ -16,90 +8,53 @@ function makingStars(val) {
-  else {
+  if (restaurantsToShow.length == 0) {
+    $('#list').html('Brak lokali w tej kategorii');
+  } else {
     $('#list').html(
-      '<table class="table locals"><th>Nazwa</th><th>Adress</th><th>Oceny</th></table>'
+      '<table class="table restaurants"><th>Nazwa</th><th>Adress</th><th>Oceny</th></table>'
     );
-    $.each(localsToShow, (i, l) => {
-      $('.locals').append(
+    $.each(restaurantsToShow, function(index, restaurant) {
+      $('.restaurants').append(
         '<tr><td>' +
-          l.name +
+          restaurant.name +
           '</td><td>' +
-          l.location.adress +
-          '</td><td><div class="stars1"><select class="stars" ><option value="1">1</option> <option value="2">2</option><option value="3">3</option><option value="4">4</option><option value="5">5</option></select></div></td></tr>'
+          restaurant.location.adress +
+          '</td><td><div class="stars1" data-toggle="tooltip" data-placement="left" title="' +
+          restaurant.rate +
+          '"><select class="stars" ><option value="1">1</option> <option value="2">2</option><option value="3">3</option><option value="4">4</option><option value="5">5</option></select></div></td></tr>'
       );
-      makingStars(l.rate);
+      initializeStarRating(restaurant.rate);
       $('.stars1').on('click', function() {
         $('#myModal').modal('show');
-        createModal(l);
+        createModal(restaurant);
       });
     });
   }
 }
 
-var id = '';
-
-$(document).ready(function() {
-  id = chceckIdCategory();
-
-  showCategory();
-});
-
-$(window).on('hashchange', function(e) {
-  id = chceckIdCategory();
-  showCategory();
-});
-
-function createModal(local) {
-  $('.ratesModalTitle').html('Oceny dla lokalu ' + local.name);
+function createModal(restaurant) {
+  $('.ratesModalTitle').html('Oceny dla lokalu ' + restaurant.name);
   $('.ratesModalBody').html(
     '<table class="table rate"><th>Uzytkownik</th><th>Ocena</th><th>Data</th><th>tres</th></table>'
   );
-  $.each(rates, (i, r) => {
-    var date = r.created_at.split('-').reverse();
-    var now = moment().format('YYYY-MM-DD').split('-');
-    var diffDays = moment(now).diff(date, 'days');
-    var diffMonths = moment(now).diff(date, 'month');
-    var diffYears = moment(now).diff(date, 'year');
-    var message = '';
-    if (diffDays == 1) {
-      message = 'wczoraj';
-    } else if (diffDays < 30) {
-      message = diffDays + ' dni temu';
-    } else if (diffMonths == 1) {
-      message = 'miesiac temu';
-    } else if (diffMonths < 12) {
-      message = diffMonths + ' miesiecy temu';
-    } else if (diffYears == 1) {
-      message = 'rok temu';
-    } else {
-      message = diffYears + ' lata temu';
-    }
+  $.each(rates, function(index, rate) {
+    var date = rate.created_at.split('-').reverse();
+    message = moment(date).toNow();
+    console.log(date);
+    console.log(moment().format('YYYY-MM-DD'));
```

Comment:
i realize this is because the feature's not ready yet and i asked you to do a PR anyway, but, please remember to avoid pushing console.logs :)